### PR TITLE
Refactor: Remove some global classes in module_elecstate.

### LINKED ISSUE #2318: We will remove part of global classes.
### Describe Current Status and Possible Solution

We will remove global classes: kv, sf, rhopw, bigpw, wfcpw, CHR_MIX, symm, wf.
They will involve modules of elecstate、hamilt_general/lcao、hamilt_pw、io、ri、elecstate、esolver

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by sunliang98**
Remove `GlobalC::kv`, `GlobalC::sf`, `GlobalC::bigpw` in `module_elecstate`.
fix #2318

### CODE REVIEW

File: source/module_elecstate/elecstate.cpp [hunk 1]
```diff
@@ -406,6 +408,9 @@ void ElecState::init_ks(Charge* chg_in, // pointer for class Charge
     // initialize ekb and wg
     this->ekb.create(nk_in, GlobalV::NBANDS);
     this->wg.create(nk_in, GlobalV::NBANDS);
+
+    this->nbz = nbz_in;
```
Comment (hongriTianqi) [inline source/module_elecstate/elecstate.cpp]: Should `nbz` and `bz` become members of elecstate? They are members of bigpw and I remember comments say that bigpw will be removed in due course. @dyzheng

**Review by hongriTianqi — CHANGES_REQUESTED** [review]

File: source/module_elecstate/elecstate.cpp (hunk 1 above)
Comment (hongriTianqi) [reply source/module_elecstate/elecstate.cpp]: Another choice is to add bigpw as the member of Elecstate directly.

**Review by hongriTianqi — CHANGES_REQUESTED** [review]

File: source/module_elecstate/test/elecstate_lcao_test.cpp
Comment (hongriTianqi) [inline source/module_elecstate/test/elecstate_lcao_test.cpp:1]: I suggest to remove this file directly.

File: source/module_elecstate/test/updaterhok_pw_test.cpp
Comment (hongriTianqi) [inline source/module_elecstate/test/updaterhok_pw_test.cpp:1]: I suggest to remove this test directly

Comment (sunliang98) [reply source/module_elecstate/test/updaterhok_pw_test.cpp:1]: OK, it's done.

File: source/module_elecstate/elecstate.cpp (hunk 1 above)
Comment (sunliang98) [reply source/module_elecstate/elecstate.cpp]: I have added bigpw as the member of Elecstate.

**Review by hongriTianqi — APPROVED** [review]

**Review by mohanchen — APPROVED** [review]

**Comment by Qianruipku** [comment]
fix #2318

**Pull request merged**